frontend/src/MainBlock/Repositories/FetchingParams/FetchingParams.js

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+
+import { withStyles } from '@material-ui/core/styles';
+import TextField from '@material-ui/core/TextField';
+import Badge from '@material-ui/core/Badge';
+import IconButton from '@material-ui/core/IconButton';
+import SortIcon from '@material-ui/icons/Sort';
+import Menu from '@material-ui/core/Menu';
+
+import connect from 'react-redux/es/connect/connect';
+import SortingAttributes from './SortingAttributes';
+
+const styles = () => ({
+    repositoriesSearchInput: {
+        width: '300px',
+        marginBottom: '10px',
+    },
+});
+
+function FetchingParams({ classes, searchString, handleSearchStringChange, handleOrderStringChange }) {
+    const [anchorEl, setAnchorEl] = useState(null);
+    const [sortedAttributes, setSortedAttributes] = useState([]);
+
+    const handleSortButtonClick = (event) => {
+        setAnchorEl(event.currentTarget);
+    };
+
+    const handleClose = () => {
```

Comment:
```suggestion
    const handleSortMenuClose = () => {
```
Кажется, такое имя будет более подходящим, потому как мы не весь компонент закрываем

Reply:
Да, спасибо